Card packages page updates (#592)
swan-io/docs: docs/topics/cards/card-packages/guide-product-configuration.mdx

---
title: Configure a card product
---

To create a new card product in the Dashboard, first select and configure the [card package](/topics/cards/card-packages/). 
Your selection determines the features, capabilities, and pricing of the card your users can access.

This guide shows you how to configure card packages on card products.

## Create a card product {#create}

1. Go to **Dashboard** > **Settings** > **Cards**.
1. Select **+ New card product**.

![Screenshot of the Dashboard modal to add a new card product](../../../images/topics/cards/configure-card-package.png)

3. Choose the **Standard**, **Essential**, or **Premium** card package.

> To offer a card with only basic features (no insurance, base spending limits, and base foreign exchange fees), select **No package**.

Reviewer: ```suggestion
Select **No package** for basic features with no insurance and base fees.
```
Main point up front.
